src/client/components/GetChannels/GetChannels.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from 'react';
+import { useParams, Link } from 'react-router-dom';
+
+function GetChannels() {
```

Comment:
If you could rename this component to `OverviewChannels`, that would be slightly better name.